docs/modules/ROOT/pages/advanced-how-tos/building-olm.adoc

```diff
@@ -0,0 +1,27 @@
+= Building an Operator Lifecycle Manager (OLM) in {ProductName}
+
+If you are using an link:https://olm.operatorframework.io/docs/[Operator Lifecycle Manager (OLM)] for your application, you can build the OLM in {ProductName}. The first procedure in this document explains how to use {ProductName} to build an OLM in the most basic sense--building its operator and bundle. The second procedure is optional but recommended. It explains how you can automatically update the image references to the operator and operands that should be present in your bundle.
+
+
+== Builing the operator and the bundle
+
+[NOTE] 
+====
+This procedure assumes that the source code for your OLM operator and bundle, including the Dockerfile, are in the same git repository, per OLM convention.
```

Comment:
If they are building an operator and a bundle, they will have two Dockerfiles at the very least, so I would change `Dockerfile` to plural here